### DISCUSSION TITLE
2.0正式版什么时候发布

### DISCUSSION THREAD

**Discussion by nine-city** [milvus-io/milvus#12752] [Q&A and General discussion] [answered]
2.0正式版什么时候发布

**Comment by yhmo** [comment, marked as answer]
Well, this is a "灵魂拷问".
Consider there are still 15 ~ 20 critical issues not yet closed, the product quality is below the release criteria, we postpone the GA version.
You can check the GA state here: https://github.com/milvus-io/milvus/milestone/28 . To release GA, we need to fix all the "**priority/critical-urgent** " and "**priority/urgent** " issues.
